### ISSUE TITLE
small change to make in column_as_rownames()

### ISSUE THREAD

**Issue by etiennebacher** [easystats/datawizard#95]
In `column_as_rownames()`, the behavior implemented in #80 is to use "rowname" as default if `var = NULL`. This doesn't work because:
https://github.com/easystats/datawizard/blob/817d90aa3bba501435cfceb55c6309ecc9b190f5/R/utils_data.R#L48-L50
comes after the checks with `is.character()` and `is.numeric()` which means that this condition is never called if `var = NULL`. All that is needed is to put the `NULL` check first, but I'm not sure whether using "rowname" if `var = NULL` should be the default behavior. 

Basically there is something to change but it depends on your decision:
* either keep this behavior but put the `NULL` check first (+ add a test for when `var = NULL`)
* remove this behavior and therefore also this condition because it is useless as is

I'm ok to do a PR for this if you want

**Comment by IndrajeetPatil** [comment]
Thanks for the detailed account of the problem.

> remove this behavior and therefore also this condition because it is useless as is

I would vote for this! This check is redundant, and can be removed.

Feel to free to make a PR, and, if you want, you can also add yourself as a contributor to `DESCRIPTION` in the same PR.
